LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2013 Shopify
+Copyright (c) 2021 Shopify
```

Comment:
I think it should be a range to indicate that it includes code that was written earlier than the current year.

```suggestion
Copyright (c) 2013-2021 Shopify
```

Reply:
Done!